.npmrc

```diff
@@ -1,8 +1,10 @@
+enable-pre-post-scripts=true
```

Comment:
Interesting addition. I didn't notice that the pre-post hooks and `simple-git-hooks` were never ran. But I guess this should be fine too.